cleverhans/attacks_tf.py

```diff
@@ -556,7 +556,7 @@ def body(x_in, y_in, domain_in, i_in, cond_in):
 
 
 def jacobian_augmentation(sess, x, X_sub_prev, Y_sub, grads, lmbda,
-                          feed=None):
+                          aug_batch_size=512, feed=None):
```

Comment:
This value could be larger than the initial substitute training set size. Could you add some logic that sets `aug_batch_size` to be the min between `aug_batch_size` and `X_sub_prev.shape[0]` to handle that case?